src/app/shared/forms/planet-rating.component.ts

```diff
@@ -141,6 +147,10 @@ export class PlanetRatingComponent implements OnChanges {
       });
   }
 
+  clearRating() {
+    console.log("button was clicked");
+  }
+
```

Comment:
I see, is there any way I can call it directly?

Reply:
i dont think so, since both are components, typically you would want to call a service to a components (for the functions), not a component to a components.